src/sdk/pynni/nni/bohb_advisor/bohb_advisor.py

```diff
@@ -573,3 +573,30 @@ def handle_report_metric_data(self, data):
 
     def handle_add_customized_trial(self, data):
         pass
+
+    def handle_feed_tuning_data(self, data):
+        """
+        Parameters
+        ----------
+        data:
+            it is an object which has keys 'parameter', 'value'
+
+        Raises
+        ------
+        AssertionError
+            data doesn't have required key 'parameter' and 'value'
+        """
+        assert "parameter" in data
```

Comment:
You'll receive a list of data, not a single line of data like `{"paramter": {"x": 0.5, "y": 0.9}, "value": 0.03}`

Reply:
ok, will change this logic.

Reply:
Done.